portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -2488,4 +2488,126 @@ definition {
 		FormFields.viewTextLabel(fieldLabel = "Text");
 	}
 
+	@description = "Access Portal Global Menu and validate CONFIGURATION string at Control Panel"
+    @priority = "5" \
+    test WriteNewForm {
+
+        FormsAdmin.addForm();
```

Comment:
Where is your TASKS? You must follow the Liferay Default, using task for help other workers to understand your code. You must do the tasks using BDD Framework, seach about BDD and do it correctly. Make tasks and send it again!